Enterprise-CMCS/eAPD
# 4500 Add MDBT Budget Tables to Proposed Budget

Resolves #4500 

### Description
Updates to the Proposed Budget tables for MMIS

### Chromatic Link
https://www.chromatic.com/pullrequest?appId=61d5b948cf6f17003a12bf77&number=4667

### Significant changes or possible side effects
Updated budget to add in keyStatePersonnel for `ddi` and `mando`

### Automated test cases written

Existing tests updated to account for new design / table / copy changes

### Steps to manually verify this change

1. Login to HITECH apd and verify existing Proposed Budget tables are the same (with one exception- the HIT/HIE/MMIS Total is now Subtotal)
2. Login to MMIS type apd
3. Verify all acceptance criteria is met and new designs are as specified according to the ticket and Figma

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when QA has


### This pull request can be merged when
